packages/commerce-sdk-react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@salesforce/commerce-sdk-react",
-  "version": "1.1.0-dev.2",
+  "version": "1.1.0-dev.3",
```

Comment:
Are we releasing this one?

Reply:
Yep, there are a few bug fixes we'd like to get out. (See details in the changelog.)